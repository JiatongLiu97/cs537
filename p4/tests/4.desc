[8] Allocate multiple pages and verify its initial state with CLOCKSIZE = 8
