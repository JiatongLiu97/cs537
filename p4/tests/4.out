XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0x46, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0x45, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0x44, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 3: pdx: 0x0, ptx: 0x43, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 4: pdx: 0x0, ptx: 0x42, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 5: pdx: 0x0, ptx: 0x41, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 6: pdx: 0x0, ptx: 0x40, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 7: pdx: 0x0, ptx: 0x3F, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 8: pdx: 0x0, ptx: 0x3E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 9: pdx: 0x0, ptx: 0x3D, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 10: pdx: 0x0, ptx: 0x3C, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 11: pdx: 0x0, ptx: 0x3B, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 12: pdx: 0x0, ptx: 0x3A, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 13: pdx: 0x0, ptx: 0x39, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 14: pdx: 0x0, ptx: 0x38, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 15: pdx: 0x0, ptx: 0x37, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 16: pdx: 0x0, ptx: 0x36, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 17: pdx: 0x0, ptx: 0x35, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 18: pdx: 0x0, ptx: 0x34, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 19: pdx: 0x0, ptx: 0x33, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 20: pdx: 0x0, ptx: 0x32, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 21: pdx: 0x0, ptx: 0x31, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 22: pdx: 0x0, ptx: 0x30, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 23: pdx: 0x0, ptx: 0x2F, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 24: pdx: 0x0, ptx: 0x2E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 25: pdx: 0x0, ptx: 0x2D, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 26: pdx: 0x0, ptx: 0x2C, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 27: pdx: 0x0, ptx: 0x2B, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 28: pdx: 0x0, ptx: 0x2A, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 29: pdx: 0x0, ptx: 0x29, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 30: pdx: 0x0, ptx: 0x28, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 31: pdx: 0x0, ptx: 0x27, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 32: pdx: 0x0, ptx: 0x26, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 33: pdx: 0x0, ptx: 0x25, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 34: pdx: 0x0, ptx: 0x24, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 35: pdx: 0x0, ptx: 0x23, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 36: pdx: 0x0, ptx: 0x22, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 37: pdx: 0x0, ptx: 0x21, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 38: pdx: 0x0, ptx: 0x20, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 39: pdx: 0x0, ptx: 0x1F, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 40: pdx: 0x0, ptx: 0x1E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 41: pdx: 0x0, ptx: 0x1D, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 42: pdx: 0x0, ptx: 0x1C, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 43: pdx: 0x0, ptx: 0x1B, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 44: pdx: 0x0, ptx: 0x1A, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 45: pdx: 0x0, ptx: 0x19, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 46: pdx: 0x0, ptx: 0x18, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 47: pdx: 0x0, ptx: 0x17, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 48: pdx: 0x0, ptx: 0x16, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 49: pdx: 0x0, ptx: 0x15, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 50: pdx: 0x0, ptx: 0x14, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 51: pdx: 0x0, ptx: 0x13, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 52: pdx: 0x0, ptx: 0x12, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 53: pdx: 0x0, ptx: 0x11, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 54: pdx: 0x0, ptx: 0x10, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 55: pdx: 0x0, ptx: 0xF, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 56: pdx: 0x0, ptx: 0xE, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 57: pdx: 0x0, ptx: 0xD, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 58: pdx: 0x0, ptx: 0xC, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 59: pdx: 0x0, ptx: 0xB, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 60: pdx: 0x0, ptx: 0xA, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 61: pdx: 0x0, ptx: 0x9, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 62: pdx: 0x0, ptx: 0x8, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 63: pdx: 0x0, ptx: 0x7, writable bit: 1, encrypted: 1, ref: 0
